Fixed a few typos and updated views guide

wiserfirst opened the pull request:
1 Fixed a few typos: were -> was
2 Updated the html in the Views guide to match the latest Phoenix version (1.0.3)
3 Updated a few words in the Views guide

lancehalvorsen commented on B_adding_pages.md:
```diff
@@ -237,7 +237,7 @@ It's good to remember that the keys to the params [Dict](http://elixir-lang.org/
 For the last piece of this puzzle, we'll need a new template. Since it is for the `show` action of the `HelloController`, it will go into the `web/templates/hello` directory and be called `show.html.eex`. It will look surprisingly like our `index.html.eex` template, except that we will need to display the name of our messenger.
 
-To do that, we'll use the special eex tags for executing Elixir expressions - `<%=  %>`. Notice that the initial tag has an equals sign like this: `<%=` . That means that any Elixir code that goes between those tags will be executed, and the resulting value will replace the tag. If the equals sign were missing, the code would still be executed, but the value would not appear on the page.
+To do that, we'll use the special eex tags for executing Elixir expressions - `<%=  %>`. Notice that the initial tag has an equals sign like this: `<%=` . That means that any Elixir code that goes between those tags will be executed, and the resulting value will replace the tag. If the equals sign was missing, the code would still be executed, but the value would not appear on the page.
```
I believe that "were" here is correct. It's the subjunctive case.

lancehalvorsen commented on C_routing.md:
```diff
@@ -46,7 +46,7 @@ Inside the scope block, however, we have our first actual route.
 The first argument to these macros is the path. Here, it is the root of the application, `/`. The next two arguments are the controller and action we want to have handle this request. These macros may also take other options, which we will see throughout the rest of this guide.
 
-If this were the only route in our router module, the clause of the `match/3` function would look like this after the macro is expanded.
+If this was the only route in our router module, the clause of the `match/3` function would look like this after the macro is expanded.
```
I believe that "were" is correct here as well - also the subjunctive case.

lancehalvorsen commented on deployment/I_exrm_releases.md:
````diff
@@ -312,7 +312,7 @@ $ rel/hello_phoenix/bin/hello_phoenix ping
 ```
 
-It's still alive and responding. This is a feature! If the server were to go down every time we stopped a remote console, that would be a problem. Note that this situation is different from when we ran the `console` command above. The `remote_console` command is not intended to start a release locally, but rather to connect to one which has already started.
+It's still alive and responding. This is a feature! If the server was to go down every time we stopped a remote console, that would be a problem. Note that this situation is different from when we ran the `console` command above. The `remote_console` command is not intended to start a release locally, but rather to connect to one which has already started.
````
This is another example of the subjunctive case.

lancehalvorsen commented:
Hi @wiserfirst. Thanks so much for this contribution. Sorry I was so late getting to it. ElixirConf and a killer cold have held me up for a few days.

I've left a few very small comments. If we can address those and squash the commits, we'll be ready to go.

Thanks again!

maxbeizer commented:
+1 to the [subjunctive case](https://en.wikipedia.org/wiki/English_subjunctive) comments.

wiserfirst commented:
@lancehalvorsen Thank you very much for you comments. Sorry for my bad grammar.

I've reverted all the updates for were to was and squashed the commits. Please review.

Thanks again!

lancehalvorsen commented:
Hi @wiserfirst, this is great! There's no need to apologize at all. The subjunctive case is a little unusual.

Thanks so much for your contribution. <3 <3 <3

wiserfirst commented:
@lancehalvorsen Really glad I can contribute :satisfied:
